.reuse/dep5

```diff
@@ -301,43 +300,48 @@ Copyright: Icons8
 License: LicenseRef-Icons8
 
 Files: libs/delaunay-triangulation/*
-Copyright: 2015 Simon Zeni
+Copyright: 2015, Simon Zeni
 License: MIT
 
 Files: libs/dxflib/*
-Copyright: 2001 Robert J. Campbell Jr., 2001-2013 RibbonSoft GmbH
+Copyright: 2001-2013, RibbonSoft GmbH
 License: GPL-2.0-or-later
 
-Files: libs/fontobene-qt/*
-Copyright: Fontobene Developers
+Files: libs/fontobene-qt5/*
+Copyright: 2012-2020, Urban Bruhin
 License: MIT or Apache-2.0
 
 Files: libs/googletest/*
-Copyright: Google and various contributors
+Copyright: 2005-2020, Google Inc and various contributors
+           2018-2019, Google LLC
 License: BSD-3-Clause
 
 Files: libs/googletest/googlemock/scripts/generator/*
 Copyright: Google and various contributors
 License: Apache-2.0
 
 Files: libs/hoedown/*
-Copyright: 2008-2014 Natacha Porté, Vicent Martí, Xavier Mendez, Devin Torres and the Hoedown authors
+Copyright: 2008-2014, Natacha Porté, Vicent Martí, Xavier Mendez, Devin Torres and the Hoedown authors
 License: ISC
 
 Files: libs/muparser/*
-Copyright: 2020 Ingo Berg
+Copyright: 2004-2020, Ingo Berg
 License: BSD-2-Clause
 
+Files: libs/polyclipping/
```

Comment:
Polyclipping is already listed below, what's wrong with it?

Reply:
Good spot, removed!